genienlp/data_utils/bootleg.py

```diff
@@ -177,19 +178,16 @@ def disambiguate_mentions(self, config_args):
 
     def post_process_bootleg_types(self, title):
         types = None
-        if title in self.type_mapping_match:
-            types = self.type_mapping_match[title]
-        else:
-            for key in self.type_mapping_include.keys():
-                if key in title:
-                    types = self.type_mapping_include[key]
+        for key in self.wiki2normalized_type.keys():
```

Comment:
It looks like wiki2normalized_type is never actually used as a dictionary. Should we make it a list of pairs instead?

Reply:
Is there an advantage to lists? dictionary seems more consistent since all the other mappings are a dictinary

Reply:
There is a marginal speed advantage. Also, it would preserve the ordering given in the json file in case there are overlapping patterns.